### PR TITLE
Change view only input with contact lookup field

### PR DESCRIPTION
**Pull request by amacar**
[ch5397]

### CODE REVIEW

File: common/v2/components/ContactLookupField.tsx [hunk 1]
```diff
@@ -1,65 +1,50 @@
...
+  findNextRecipientLabel,
+  getBaseAssetByNetwork,
+  AssetContext
+} from 'v2/services/Store';
 import { isValidETHAddress, isValidENSName } from 'v2/services/EthService';
 import { useEffectOnce } from 'v2/vendor';
+import UnstoppableResolution from 'v2/services/UnstoppableService';
 
 import ContactLookupDropdown from './ContactLookupDropdown';
+import { isValidETHRecipientAddress } from 'v2/services/EthService/validators';
```
[the remark this answers is not in the document]
Comment (amacar) [reply common/v2/components/ContactLookupField.tsx]: Done, thanks for noticing that.

**Comment by amacar** [comment]
> Nice! Can you possibly make the base account label change from "Recipient _x_" to "View Only Account _x_" or something like that instead? That'll provide some distinction. It also seems that after you type in an address and it labels it with "Recipient _x_", it relabels it as "View Only Account _x_" when you continue on to the dashboard. Is that intentional?

This is intentional for the same reason that you wrote. When using the ContactLookup component are addresses saved as 'Recipient X' and ens domains as 'Domain_name_till_the_first_dot' and that is why they are relabeled when adding an account (to remove confusion when view only account would be labeled as 'Recipient X' or as 'Domain name').
If I change the base account label from 'Recipient' to 'View Only' then we have another "problem" where the user cancels add view-only account flow but the label 'View Only X' still remains saved in the contact book which.
What do you think would be the best behavior?

**Comment by blurpesec** [comment]
> This is intentional for the same reason that you wrote. When using the ContactLookup component are addresses saved as 'Recipient X' and ens domains as 'Domain_name_till_the_first_dot' and that is why they are relabeled when adding an account (to remove confusion when view only account would be labeled as 'Recipient X' or as 'Domain name').
> If I change the base account label from 'Recipient' to 'View Only' then we have another "problem" where the user cancels add view-only account flow but the label 'View Only X' still remains saved in the contact book.
> What do you think would be the best behavior?

This is fine.

**Comment by amacar** [comment]
> ~This is fine.~ The way it is currently is fine. c: This ready for merge? :-)
Yes, it is ready for a merge if there are no other requested changes.

**Pull request merged**